### PR TITLE
Reject CVE-2022-29175

### PR DESCRIPTION
**Pull request by shelbyc**
Include message to indicate this is not actually a security issue. See the [maintainer advisory](https://github.com/vyperlang/vyper/security/advisories/GHSA-42j8-8cjv-j5r9) and [maintainer's issue on the topic](https://github.com/vyperlang/vyper/issues/2845) for more information.

### CODE REVIEW

File: 2022/29xxx/CVE-2022-29175.json [hunk 1]
```diff
@@ -1,32 +1,8 @@
 {
```
Comment (cve-team) [inline 2022/29xxx/CVE-2022-29175.json]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/29xxx/CVE-2022-29175.json]: ('Invalid control character at: line 4 column 32 (char 110)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/29xxx/CVE-2022-29175.json (hunk 1 above)
Comment (shelbyc) [reply 2022/29xxx/CVE-2022-29175.json]: Replaced smart quote with standard quote

Comment (shelbyc) [reply 2022/29xxx/CVE-2022-29175.json]: Replaced smart quote with standard quote

**Pull request merged**